src/history/test/SerializeTests.cpp

```diff
@@ -12,23 +12,40 @@ using namespace stellar;
 
 TEST_CASE("Serialization round trip", "[history]")
 {
-    std::vector<std::string> testFiles = {
-        "stellar-history.testnet.6714239.json",
-        "stellar-history.livenet.15686975.json"};
-    for (auto const& fn : testFiles)
+    std::vector<std::string> inputFiles = {
+        "stellar-history.testnet.6714239.input.json",
+        "stellar-history.livenet.15686975.input.json",
+        "stellar-history.testnet.6714239.networkPassphrase.json"};
+    std::vector<std::string> outputFiles = {
```

Comment:
you should just add some prefix string to the output file name (so you don't need this vector) and that way we have a clear mapping between files.

Reply:
also, I think files should be named `expected-output-XYZ.json` - output is a lie :)

Reply:
(and yes this will cause the one with the passphrase to be duplicated but that's fine)

Reply:
Renamed the file to `expected-output-...`.

After adding the logic to print `networkPassphrase` only if it's non-empty, the expected output file is only necessary when the _current_ code reads a file _with_ `networkPassphrase` and output a file _without_ `networkPassphrase`. In all other cases, the input and expected output are the same. So I decided to keep these two vectors in order to avoid copying a lot of input files. Let me know what you think!